ambry-quota/src/main/java/com/github/ambry/quota/AmbryQuotaManager.java

```diff
@@ -57,36 +62,52 @@ public AmbryQuotaManager(QuotaConfig quotaConfig, ThrottlePolicy throttlePolicy,
     }
     this.throttlePolicy = throttlePolicy;
     this.quotaConfig = quotaConfig;
+    this.quotaMetrics = new QuotaMetrics(metricRegistry);
   }
 
   @Override
   public void init() {
+    Timer.Context timer = quotaMetrics.quotaManagerInitTime.time();
     for (QuotaEnforcer quotaEnforcer : requestQuotaEnforcers) {
       quotaEnforcer.init();
     }
+    timer.stop();
   }
 
   @Override
   public ThrottlingRecommendation getThrottleRecommendation(RestRequest restRequest) {
+    Timer.Context timer = quotaMetrics.quotaEnforcementTime.time();
```

Comment:
please move this line after if statement below.

Reply:
done.